source/getting-started/faq.rst

```diff
@@ -150,4 +150,9 @@ We therefore recommend **Docker Desktop as the official support** for Exegol.
 
 We do **not** guarantee wrapper stability with a directly installed WSL docker.
 
+How to retrieve your desktop login details
```

Comment:
I would make it a question, like the others: `How to retrieve the desktop login details?`